servers/src/test/resources/JournalEntries.json

```diff
@@ -0,0 +1,248 @@
+{
```

Comment:
I think this file should be called "JsonJournalEntries.json" or something, since this is only used for the "JsonJournalFormatterTest".